src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let RESULT = 0;
+  const adder = (...args) => {
+    if (args.length === 0) {
+      const value = RESULT;
+
+      RESULT = 0;
+
+      return value;
+    } else {
```

Comment:
```suggestion
    }
```
else is redundant here